src/main/java/com/revature/models/TripDTO.java

```diff
@@ -1,27 +1,52 @@
 package com.revature.models;
 
 import java.time.LocalDateTime;
+import java.util.ArrayList;
 import java.util.List;
 
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 @NoArgsConstructor @Data
 public class TripDTO {
+
 	private int tripId;
+
+	@NotNull
+	@Pattern(regexp = "[a-zA-Z0-9 ]+", message = "Trip names may only contain letters, numbers, and spaces")
 	private String name;
-	private User driver;
-	private List<User> riders;
+
+	@NotNull
+	private UserDTO driver;
+
+	private List<UserDTO> riders;
```

Comment:
This should probably be annotated with @ Valid. That will enforce validation on every element within the list.